webapp/apifuncs.py

```diff
@@ -59,22 +61,16 @@ def get_pitching_stats(plyr_data, key, team):
     """
     :param plyr_data: player entry from player master file
     :param key:       player key used in result dictionary
-    :param team:      team master entry for player's team, or could be None
+    :param team:      team master entry for player's team
 
     formats the pitchers dictionary for player entry used in html form
     """
 
```

Comment:
Don't need any more since Home or Away is already passed to routine and part of team master now